metron-deployment/roles/metron_streaming/defaults/main.yml

```diff
@@ -0,0 +1,81 @@
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+---
+metron_directory: /usr/metron/{{ metron_version }}
+metron_solr_jar_name: metron-solr-{{ metron_version }}.jar
+metron_elasticsearch_jar_name: metron-elasticsearch-{{ metron_version }}.jar
+metron_parsers_jar_name: metron-parsers-{{ metron_version }}.jar
+
+metron_common_bundle_name: metron-common-{{ metron_version }}-archive.tar.gz
+metron_data_management_bundle_name: metron-data-management-{{ metron_version }}-archive.tar.gz
+metron_enrichment_bundle_name: metron-enrichment-{{ metron_version }}-archive.tar.gz
+metron_solr_bundle_name: metron-solr-{{ metron_version }}-archive.tar.gz
+metron_elasticsearch_bundle_name: metron-elasticsearch-{{ metron_version }}-archive.tar.gz
+metron_parsers_bundle_name: metron-parsers-{{ metron_version }}-archive.tar.gz
+metron_common_bundle_path: "{{ playbook_dir }}/../../metron-platform/metron-common/target/{{ metron_common_bundle_name }}"
+metron_data_management_bundle_path: "{{ playbook_dir }}/../../metron-platform/metron-data-management/target/{{ metron_data_management_bundle_name }}"
+metron_enrichment_bundle_path: "{{ playbook_dir }}/../../metron-platform/metron-enrichment/target/{{ metron_enrichment_bundle_name }}"
+metron_solr_bundle_path: "{{ playbook_dir }}/../../metron-platform/metron-solr/target/{{ metron_solr_bundle_name }}"
+metron_elasticsearch_bundle_path: "{{ playbook_dir }}/../../metron-platform/metron-elasticsearch/target/{{ metron_elasticsearch_bundle_name }}"
+metron_parsers_bundle_path: "{{ playbook_dir }}/../../metron-platform/metron-parsers/target/{{ metron_parsers_bundle_name }}"
+
+
+config_path: "{{ metron_directory }}/config"
+zookeeper_config_path: "{{ config_path }}/zookeeper"
+zookeeper_global_config_path: "{{ zookeeper_config_path }}/global.json"
+metron_solr_properties_config_path: "{{ metron_directory }}/config/solr.properties"
+metron_elasticsearch_properties_config_path: "{{ metron_directory }}/config/elasticsearch.properties"
+metron_parsers_properties_config_path: "{{ metron_directory }}/config/parsers.properties"
+hbase_config_path: "/etc/hbase/conf"
+hdfs_config_path: "/etc/hadoop/conf"
+
+threat_intel_bulk_load: True
+threat_intel_bin: "{{ metron_directory }}/scripts/threatintel_bulk_load.sh"
+threat_intel_work_dir: /tmp/ti_bulk
+threat_intel_csv_filename: "threat_ip.csv"
+threat_intel_csv_filepath: "../roles/metron_platform/templates/{{ threat_intel_csv_filename }}"
```

Comment:
The threat intel csv file is still at ../roles/metron_streaming/templates, probably okay there, but this will need to be changed (or the file moved, dealer's choice).